src/app/archive/archive.ml

```diff
@@ -1,14 +1,28 @@
 open Core
+open Async
+open Coda_base
+open Archive_lib
 
-module Get_state_hash =
-[%graphql
-{|
-query get_state_hash {
-  blocks {
-    state_hash
-  }
-}
-
-|}]
-
-let () = printf !"Hello Coda"
+let () =
+  let keys = Array.init 2 ~f:(fun _ -> Signature_lib.Keypair.create ()) in
+  let user_command =
+    Quickcheck.random_value
+      (User_command.Gen.payment_with_random_participants ~keys
+         ~max_amount:10000 ~max_fee:1000 ())
+  in
+  let block_time =
+    Block_time.of_span_since_epoch @@ Block_time.Span.of_ms (Int64.of_int 100)
+  in
+  let graphql_obj = Types.User_command.encode user_command block_time in
+  let graphql =
+    Graphql_commands.Transaction_pool_insert.make
```

Comment:
I thought we said txn pool wouldn’t be part of archive node?

Reply:
We can still apply transaction pool diffs though. For our discussion yesterday, we reached to a consensus that for transaction status, we will remove the `Schedule` enum and all transactions that are not associated with a block are unknown.  We will also remove the field `Transaction.transaction_pool_membership`